aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Learn how to use the ASP.NET Core SignalR HubContext service for sending notifications to clients from outside a hub.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+# Send messages from outside a hub
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+
+The SignalR hub is the core abstraction for sending messages to clients connected to the SignalR server. It's also possible to send messages from other places in your app using the `IHubContext<T>` service. This article explains how to access a SignalR `IHubContext<T>` to send notification to clients from outside a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Get an instance of `HubContext`
+
+In ASP.NET Core SignalR, you can access an instance of `IHubContext<T>` via dependency injection. You can inject an instance of `IHubContext<your_hub_type>` into a controller, middleware, or other DI service. Use the instance to send messages to clients.
```

Comment:
> <your_hub_type>

Is this the recommended way for writing this @scottaddie ?

Reply:
It's more obvious to the reader that this is a placeholder when you write it this way.

Reply:
Cool 👍

Reply:
I don't know that I like that syntax... we generally still want to keep things looking my .NET types. I'd prefer something like `MyHub`. Do we have precedent for this kind of syntax in our docs?

Reply:
I'm fine with `MyHub`, so long as it's made clear this should be replaced. Thoughts on including a `MyHub` class in the sample app? This would solve the problem too.

Reply:
Looking at [logging](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-2.1&tabs=aspnetcore2x), let's just drop the `T` altogether in the prose. The code samples will use it with the type of the real hub in the sample, so it'll be pretty obvious.